src/main/java/application/App.java

```diff
@@ -39,13 +40,22 @@ private static Router createRouter() {
 
 
         app.post("/echo_body", (Request request, Response response) -> {
-            response.successfulPost(request.getRoute());
+            response.sendBody(request.getBody().getBytes(), request.getContentFileType());
+            response.setStatus(StatusCode.ok);
         });
 
         app.get("/method_options", (Request request, Response response) -> {});
 
         app.head("/method_options", (Request request, Response response) -> {});
 
+        app.get("/method_options2", (Request request, Response response) -> {});
```

Comment:
What are all of the `/method_options2` endpoints for?

Reply:
this is to complete the acceptance tests from http_server_spec.